<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>534</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>116</width>
    <height>0</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>UpArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>文本处理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="textprocess.qrc">
    <normaloff>:/menuBar/icon/ico.png</normaloff>:/menuBar/icon/ico.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <stylestrategy>NoAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="flie_menu">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open_file_action"/>
    <addaction name="save_file_action"/>
    <addaction name="exit_action"/>
   </widget>
   <widget class="QMenu" name="tools_menu">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="copy_action"/>
    <addaction name="cut_action"/>
    <addaction name="paste_action"/>
    <addaction name="intersection_action"/>
    <addaction name="union_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="about_action"/>
    <addaction name="help_action"/>
    <addaction name="author_action"/>
   </widget>
   <addaction name="flie_menu"/>
   <addaction name="tools_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <action name="open_file_action">
   <property name="icon">
    <iconset>
     <normaloff>:/menuBar/textprocess.qrc</normaloff>:/menuBar/textprocess.qrc</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="save_file_action">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="help_action">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
  <action name="author_action">
   <property name="text">
    <string>作者</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="copy_action">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="cut_action">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="paste_action">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionA_4">
   <property name="text">
    <string>a</string>
   </property>
  </action>
  <action name="intersection_action">
   <property name="text">
    <string>交集</string>
   </property>
  </action>
  <action name="union_action">
   <property name="text">
    <string>并集</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="textprocess.qrc"/>
 </resources>
 <connections/>
</ui>
